core/src/emuwidget.cpp

```diff
@@ -44,7 +44,7 @@ void EmuWidget::init()
 	Preferences *p = Preferences::GetInstance();
 	p->init("iio_emu_path", QCoreApplication::applicationDirPath());
 
-	m_emuPath = findEmuPath();
+	m_emuPath = (startIioEmuProcess("iio-emu")) ? "iio-emu" : findEmuPath();
```

Comment:
Can you use if/else for both these statements instead of conditionals ? It makes the code a little hard to follow.